qcodes_contrib_drivers/drivers/Attocube/ANC350.py

```diff
@@ -0,0 +1,520 @@
+﻿from qcodes import Instrument, Parameter
+from qcodes.instrument.channel import InstrumentChannel, ChannelList
+from qcodes.utils.validators import Numbers
+from qcodes_contrib_drivers.drivers.Attocube.ANC350Lib import ANC350LibActuatorType, ANC350v3Lib, ANC350v4Lib
+from typing import Any, Callable, Dict, Optional, Union
+
+
+class ANC350OutputParameter(Parameter):
```

Comment:
Qcodes already supports callable get_cmd and set_cmd, which are wrapped in get_raw and set_raw, respectively. So, I am not sure why there is a need for this class.

Reply:
Yes, I know, but I didn't manage to make the optional keyword argument (`auto_off`) working. The problem was, that the `set_cmd` is wrapped by `qcodes.utils.command.Command` which doesn't forward the keyword-arguments.

Reply:
This class is used further down in "output" in line 133 and nowhere else (correct me if I'm wrong). For setting that parameter, you have used set_cmd=self._set_output, which is defined in line 429. This function itself takes auto_off argument. So, I am wondering what happens if you remove param_type=ANC350OutputParameter from "output". What kind of error message do you get?

Reply:
@FarBo I am not sure if that is possible when set takes an extra argument. Look at the code here which wraps `set_cmd` https://github.com/QCoDeS/Qcodes/blob/master/qcodes/instrument/parameter.py#L1126

However, I would suggest that it perhaps makes more sense for auto_off to be a parameter that is set independetly of the set command?

Reply:
I've found another way to realize the output-parameter. It now is a normal Parameter, with three instead of two possible values: "on", "off" and "auto"

Reply:
Sounds good. Could you please share the manual of the instrument as well?

Reply:
Yes, I've uploaded it to the main conversation: https://github.com/QCoDeS/Qcodes_contrib_drivers/pull/74#issuecomment-712824839